ompi/errhandler/errhandler.h

```diff
@@ -315,7 +338,20 @@ struct ompi_request_t;
     return MPI_SUCCESS; \
   }
 
-
+/* Same as OMPI_ERRHANDLER_RETURN for non-handle attached errors */
+#define OMPI_ERRHANDLER_NOHANDLE_RETURN(rc, err_code, message) \
```

Comment:
This could have been written as
```c 
#define OMPI_ERRHANDLER_NOHANDLE_RETURN(rc, err_code, message) \
do { \
    OMPI_ERRHANDLER_NOHANDLE_CHECK(rc, err_code, message); \
    return MPI_SUCCESS; \
} while(0);
```